app/views/admin/roles/edit.html.erb

```diff
@@ -1,21 +1,34 @@
-<% page_title "Edit #{@role.name}"  %>
-<div class="row">
-  <section class="col-md-8">
-    <h1>Edit role</h1>
-    <p class="warning">Warning: changes to roles appear instantly on the live site.</p>
+<% content_for :page_title, "Edit #{@role.name}" %>
+<% content_for :title, "Edit #{@role.name}" %>
+<% content_for :title_margin_bottom, 6 %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Edit role",
+      heading_level: 1,
+      margin_bottom: 6
+    } %>
+
+    <%= render "govuk_publishing_components/components/warning_text", {
+      text: "Changes to roles appear instantly on the live site."
+    } %>
     <%= render partial: "form", locals: {role: @role, show_instantly_live_warning: true} %>
-  </section>
-  <div class="col-md-4">
-    <%= sidebar_tabs appointments: "Appointments", govspeak_help: "Help" do |tabs| %>
-      <%= tabs.pane id: "appointments" do %>
-        <h1>Appointments</h1>
-        <p><%= link_to "New appointment", new_admin_role_role_appointment_path(@role, make_current: true), class: "btn btn-default" %></p>
-        <%= render partial: "admin/role_appointments/list", locals: {role_appointments: @role.role_appointments} %>
-        <p><%= link_to "Add previous appointment", new_admin_role_role_appointment_path(@role), class: "btn btn-default" %></p>
-      <% end %>
-      <%= tabs.pane class: "govspeak_help", id: "govspeak_help" do %>
-        <%= render partial: "admin/editions/legacy_govspeak_help" %>
-      <% end %>
-    <% end %>
+
+    <%= render "govuk_publishing_components/components/tabs", {
+      tabs: [
+        {
+          id: "appointments",
+          label: "Appointments",
+          content:
+            render(partial: "admin/roles/appointments_tab", locals: { role: @role })
+        },
+        {
+          id: "help",
+          label: "Help",
+          content: render(partial: "admin/editions/legacy_govspeak_help")
```

Comment:
This should render the GOVUK Design system formatting help

```suggestion
          content: simple_formatting_sidebar
```

Reply:
✅  nice! Thanks